src/lm_buddy/constants.py

```diff
@@ -0,0 +1,7 @@
+import os
+from pathlib import Path
+
+DEFAULT_STORAGE_PATH: str = os.getenv(
```

Comment:
Are we happy introducing our own `LM_BUDDY_STORAGE` env var for global control over outputs produced by our functions? It didn't feel right using the `RAY_STORAGE` path, since this code may not even be running on Ray.

Reply:
Maybe I'll just leave the defaults to `/tmp` for the `storage_path` on the configs. That would be fine for now too.

Reply:
I think for now let's leave it. We might eventually have staging and deployed environments, we can refactor this at that time.